plugins/phonenumber/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2025 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+# Component: PHONENUMBER
+
+#####################################################
+# Description:
+# ------------
+#    Cmake Input File for PHONENUMBER
+#    Api for google libphonenumber
+#####################################################
+
+project(phonenumber)
+
+if(PHONENUMBER)
```

Comment:
Is this ever true?  Other plugins have a USE_XX test.

Reply:
This is correct.  The general pattern for plugins is `INCLUDE_XXX` or `SUPPRESS_XXX`

Reply:
So the USE_XX test is generally if there's a third party library that we sometimes couldn't support.

If you look at h3, kafka, or mysqlembed, they all use the IF(PLUGINNAME) style.  If you look at Memcached, it uses the IF(PLUGINNAME AND USE_XX) style. I believe that was because we wanted to turn off memcached on certain build targets because it wasn't supported.  Now that we have vcpkg and bundle dependent libraries, things are simpler and we don't really need to have the USE_XX style.